plugins/android-boot/android-boot.quirk

```diff
@@ -11,3 +11,17 @@ Flags = updatable,signed-payload
 Vendor = SHIFT GmbH
 VendorId = eco.shift
 AndroidBootVersionProperty = androidboot.abl.revision
+
+# SHIFTphone 8 ABL A
+[DRIVE\UUID_0072779b-8343-75c6-525f-44ff0e1d04db&LABEL_abl-a]
+Flags = updatable,signed-payload
+Vendor = SHIFT GmbH
+VendorId = eco.shift
```

Comment:
The vendor id is normally subsystem-id:vendor-id, so maybe "DRIVE:SHIFT"?

Reply:
This unfortunately did not work, am i missing something?

`[DRIVE:SHIFT\UUID_0072779b-8343-75c6-525f-44ff0e1d04db&LABEL_abl-a]`

With the commit as-is:
```
# fwupdmgr get-devices
WARNING: UEFI capsule updates not available or enabled in firmware setup
See https://github.com/fwupd/fwupd/wiki/PluginFlag:capsules-unsupported for more information.
WARNING: This package has not been validated, it may not work properly.
shift SHIFT SHIFTphone 8
│
├─KMJS9001RM-BG01:
│     Device ID:          38c11cd05c6cd47879f3b1173e68f04b52015c84
│     Summary:            UFS device
│     Current version:    0500
│     Vendor:             SAMSUNG (SCSI:SAMSUNG)
│     Serial Number:      SAMSUNG
│     GUIDs:              e56797c0-a400-529f-a473-eba1f1a82ed7 ← SCSI\VEN_SAMSUNG&DEV_KMJS9001RM-BG01
│                         ce4bd73c-4cc4-576d-b861-2eb47b2479ae ← SCSI\VEN_SAMSUNG&DEV_KMJS9001RM-BG01&REV_0500
│     Device Flags:       • Internal device
│                         • Updatable
│   
└─abl a:
      Device ID:          726f01ecc8a79cc9e05091ea825e4c304c936eef
      Summary:            Android Bootloader
      Current version:    6.0.0
      Vendor:             SHIFT GmbH (eco.shift)
      Serial Number:      31090a61
      GUIDs:              32df84b5-af5f-512d-afbe-61a83b282962 ← DRIVE\UUID_0072779b-8343-75c6-525f-44ff0e1d04db
                          67c7794d-af25-565d-b5ce-9a42da0e93e8 ← DRIVE\UUID_0072779b-8343-75c6-525f-44ff0e1d04db&LABEL_abl-a
                          3943aaee-852f-5260-ba36-de41cc2bc863 ← DRIVE\UUID_0072779b-8343-75c6-525f-44ff0e1d04db&LABEL_abl-a&SLOT_a
      Device Flags:       • Internal device
                          • Updatable
                          • System requires external power source
                          • Supported on remote server
                          • Needs a reboot after installation
                          • Cryptographic hash verification is available
                          • Signed Payload
```

With `DRIVE:SHIFT` the `abl` device is gone:

```
# fwupdmgr get-devices
WARNING: UEFI capsule updates not available or enabled in firmware setup
See https://github.com/fwupd/fwupd/wiki/PluginFlag:capsules-unsupported for more information.
WARNING: This package has not been validated, it may not work properly.
shift SHIFT SHIFTphone 8
│
└─KMJS9001RM-BG01:
      Device ID:          38c11cd05c6cd47879f3b1173e68f04b52015c84
      Summary:            UFS device
      Current version:    0500
      Vendor:             SAMSUNG (SCSI:SAMSUNG)
      Serial Number:      SAMSUNG
      GUIDs:              e56797c0-a400-529f-a473-eba1f1a82ed7 ← SCSI\VEN_SAMSUNG&DEV_KMJS9001RM-BG01
                          ce4bd73c-4cc4-576d-b861-2eb47b2479ae ← SCSI\VEN_SAMSUNG&DEV_KMJS9001RM-BG01&REV_0500
      Device Flags:       • Internal device
                          • Updatable
```

Reply:
Nevermind that, sorry, i got confused for a second :sweat_smile:

Reply:
Should i update this for the existing quirks for the SHIFT6mq as well while at it?

Reply:
Ohh yes please! They sneaked past a previous review.